### PR TITLE
Feature/toppology

### PR DESCRIPTION
**Pull request by rayyan224**

### CODE REVIEW

File: src/components/Topology/types.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import { ReactNode } from 'react';
+import { eColorTypes } from '../../utils/getColor';
+
+export interface TypographyProps {
+    /**
+     * The tooltip ID will be generated if not assigned
+     */
+    type?: eColorTypes;
```
Comment (locothedev) [inline src/components/Topology/types.ts]: hehe copy & paste

File: src/components/Topology/Typgoraphy.styles.ts [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+import fonts from '../../styles/fonts';
+import { ITypographyFontStyle, ITypographyFontType } from './types';
+
+interface ITopologyStyledProps {
+    color: string;
+    fontStyle: ITypographyFontStyle;
+    fontType: ITypographyFontType;
+}
+
+export const StyledBlock = styled.div<ITopologyStyledProps>`
```
Comment (attilacsanyi) [inline src/components/Topology/Typgoraphy.styles.ts:11]: Minor, but shouldn't be `StyleDiv` instead due to [naming rules](https://github.com/web3ui/web3uikit/blob/master/CONTRIBUTE.md#css-naming-an-element)? Thx

File: src/components/Topology/Typgoraphy.styles.ts [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    & > h5 {
+        ${fonts.h5}
+    }
+    color: ${(props) => props.color};
+    font-weight: ${(props) => props.fontType};
+    font-style: ${(props) => props.fontStyle};
+`;
+StyledBlock.displayName = 'Tooltip';
+
+export const Styledh1 = styled.h1<ITopologyStyledProps>`
```
Comment (attilacsanyi) [inline src/components/Topology/Typgoraphy.styles.ts:34]: Naming: `StyledH1`

File: src/components/Topology/Typgoraphy.styles.ts [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+export const Styledh5 = styled.h5<ITopologyStyledProps>`
+    ${fonts.text}
+    ${fonts.h5}
+    color: ${(props) => props.color};
+    font-weight: ${(props) => props.fontType};
+    font-style: ${(props) => props.fontStyle};
+`;
+Styledh5.displayName = 'Typogography.h5';
+
+export const Styledtext = styled.p<ITopologyStyledProps>`
```
Comment (attilacsanyi) [inline src/components/Topology/Typgoraphy.styles.ts:78]: Naming: `ParagraphStyled` I know minor, just about to learn the naming convention of `web3uikit`

Comment (rayyan224) [reply src/components/Topology/Typgoraphy.styles.ts:78]: Totally agree bro, made this before that came out hehe

Comment (attilacsanyi) [reply src/components/Topology/Typgoraphy.styles.ts:78]: You were so quick, awesome :)

**Pull request closed without merging**